[1,"Error occurred during cleanup: Error: El proceso no tiene acceso al archivo porque est√° siendo utilizado por otro proceso. (os error 32): remove 'C:\\Users\\Usuario\\AppData\\Local\\Temp\\quarto-sessionc7f13ea3941cf42a'\n"]
